android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1266,6 +1295,17 @@ private ArrayList<VideoTrack> getVideoTrackInfo() {
         return videoTracks;
     }
 
+    private int getScreenShortestSide(ThemedReactContext context) {
+        DisplayMetrics displayMetrics = new DisplayMetrics();
```

Comment:
looks good, but I also wondering if we should limit / or allow to limit the size to video area resolution instead of screen resolution. 
This can be done later, but maybe we should move setLimitMaxResolution to have a string parameter to allow:
limitMaxResolution='screen' or limitMaxResolution='videoArea'

wdyt ?

Reply:
Good idea 💡